File: pyxel/core/src/audio.cc
#include <cstddef>

#include "pyxelcore/audio.h"

namespace pyxelcore {

Audio::Audio() {}

Audio::~Audio() {}

void* Audio::sound(int snd, int system) {
  return NULL;
}

void* Audio::music(int msc) {
  return NULL;
}

void Audio::play(int ch, int snd, int loop) {}

void Audio::playm(int msc, int loop) {}

void Audio::stop(int ch) {}

}  // namespace pyxelcore

File: pyxelcore/src/pyxelcore/audio.nim
#[
#
# Audio
#
void* sound(int snd, int system);
void* music(int msc);
int play_pos(int ch);
void play1(int ch, int snd, int loop);
void play(int ch, int* snd, int snd_length, int loop);
void playm(int msc, int loop);
void stop(int ch);
]#
